src/main/java/bio/terra/common/retry/CompositeBackOffPolicy.java

```diff
@@ -1,9 +1,8 @@
 package bio.terra.common.retry;
 
 import bio.terra.common.db.DatabaseRetryUtils;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit: we don't use `*` here. There is an IntelliJ setting to force it not to generate them: `Preferences > Editor > Code Style > Java` and then set very large numbers for the class count and name count.